smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/endpoints/ResolveEndpointMiddlewareGenerator.kt

```diff
@@ -31,6 +31,7 @@ import software.amazon.smithy.rulesengine.traits.StaticContextParamsTrait
 class ResolveEndpointMiddlewareGenerator(
     private val ctx: ProtocolGenerator.GenerationContext,
     private val writer: KotlinWriter,
+    private val renderPostResolution: () -> Unit = {},
 ) {
```

Comment:
**Question**: Where is a non-default argument passed for `renderPostResolution`? I couldn't find anywhere in this PR or the **aws-sdk-kotlin** one...

Reply:
The default implementation on the `ProtocolGenerator` method does